tool/src/main/java/org/wildfly/security/tool/FileSystemEncryptRealmCommand.java

```diff
@@ -0,0 +1,761 @@
+                    .build();
+            FileSystemRealmUtil.createEncryptedRealmFromUnencrypted(oldFileSystemRealm, newFileSystemRealm);
+        }
+    }
+
+    /**
+     * Creates the script/commands the user must run for Elytron to recognize
+     * and use the new filesystem-realm
+     */
+
+    private void createWildFlyScript() throws Exception {
+        int counter = 0;
+        for (Descriptor descriptor : descriptors) {
+            String outputRealmLocation = descriptor.getOutputRealmLocation();
+            String fileSystemRealmName = descriptor.getFileSystemRealmName();
+            String credentialStore = descriptor.getCredentialStore();
+            String secretKeyAlias = descriptor.getSecretKeyAlias();
+            int levels = descriptor.getLevels();
+
+            if(secretKeyAlias == null) {
+                secretKeyAlias = "key";
+            }
+            String createScriptCheck = "";
+            if (Paths.get(String.format("%s.sh", fileSystemRealmName)).toFile().exists()) {
+                createScriptCheck = prompt(
+                        false,
+                        null,
+                        false,
+                        ElytronToolMessages.msg.shouldFileBeOverwritten(String.format("%s.sh", fileSystemRealmName))
+                );
+            }
+            String fullOutputPath;
+            if (outputRealmLocation.startsWith(".")) {
+                fullOutputPath = Paths.get(outputRealmLocation.substring(2)).toAbsolutePath().toString();
+            } else {
+                fullOutputPath = Paths.get(outputRealmLocation).toAbsolutePath().toString();
+            }
+
+            if (summaryMode) {
+                summaryString.append(String.format("Configured script for WildFly named %s.sh at %s.", fileSystemRealmName, fullOutputPath));
+                summaryString.append(System.getProperty("line.separator"));
+                summaryString.append("The script is using the following names:");
+                summaryString.append(System.getProperty("line.separator"));
+                summaryString.append(String.format("Name of filesystem-realm: %s", fileSystemRealmName));
+                summaryString.append(System.getProperty("line.separator"));
+            }
+
+            List<String> scriptLines = Arrays.asList(
+                String.format("/subsystem=elytron/secret-key-credential-store=%s:add(path=%s)", "mycredstore"+counter, credentialStore),
+                String.format("/subsystem=elytron/filesystem-realm=%s:add(path=%s, levels=%s, credential-store=%s, secret-key=%s)", fileSystemRealmName, fullOutputPath+'/'+fileSystemRealmName, levels, "mycredstore"+counter, secretKeyAlias)
+            );
+
+            if (!createScriptCheck.equals("y") && !createScriptCheck.equals("yes")) {
+                Files.write(Paths.get(String.format("%s/%s.sh", outputRealmLocation, fileSystemRealmName)), scriptLines, StandardOpenOption.CREATE, StandardOpenOption.TRUNCATE_EXISTING);
```

Comment:
Why `.sh`? It's not a shell script. Something like `.cli` would be more descriptive.